.travis.yml

```diff
@@ -0,0 +1,11 @@
+---
+# use faster container
+sudo: false
+language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+  - openjdk6
+  - openjdk7
+# Fixes GPG dependency issues
+install: mvn install -DskipTests=true -Dgpg.skip=true
```

Comment:
http://stackoverflow.com/questions/14825039/suppressing-gpg-signing-for-maven-based-continous-integration-builds-travis-ci